spec/unit/network/authconfig_spec.rb

```diff
@@ -107,3 +107,33 @@
     end
   end
 end
+
+describe Puppet::Network::AuthConfig do
+  after :all do
```

Comment:
I think you want `after :each` otherwise state from one example can leak into another example